pkg/image/prune/prune.go

```diff
@@ -799,13 +823,14 @@ func (p *pruner) Prune(imagePruner ImageDeleter, streamPruner ImageStreamDeleter
 
 	prunableImageNodes, prunableImageIDs := calculatePrunableImages(p.g, imageNodes)
 	graphWithoutPrunableImages := subgraphWithoutPrunableImages(p.g, prunableImageIDs)
-	prunableLayers := calculatePrunableLayers(graphWithoutPrunableImages)
+	prunableConfigs, prunableLayers := calculatePrunableImageComponents(graphWithoutPrunableImages)
```

Comment:
I'm not sure why you need to separate lists if both are deleted in exactly the same list.